src/components/TarkovGraph/TarkovGraph.tsx

```diff
@@ -95,21 +123,11 @@ export function TarkovGraph({
     const elements = CytoscapeComponent.normalizeElements({nodes, edges});
 
     const onNodeClick = (event: Cytoscape.EventObject) => {
-        event.cy.edges().removeClass('highlighted'); // Remove the highlighted class from all edges
-
-        const node = event.target;
-        const allEdges = node.incomers('edge');
-
-        const highlightEdges = (edges: Cytoscape.EdgeCollection) => {
-            edges.addClass('highlighted');
-            edges.forEach((edge) => {
-                const sourceNode = edge.source();
-                const transitiveEdges = sourceNode.incomers('edge');
-                highlightEdges(transitiveEdges);
-            });
-        };
-
-        highlightEdges(allEdges);
+        const taskId = event.target[0].id();
```

Comment:
If there were multiple targets in the click event somehow, maybe we should handle all of them.